### ISSUE TITLE
Multitrack conference can't handle network changes

### ISSUE THREAD

**Issue by MaZZly** [ant-media/Ant-Media-Server#6427]
### Short description

When switching between wifi and mobile network on a mobile device connected to a multitrack conference room, streams do not start up again and stream(s) stay black or frozen.

### Environment

https://test.antmedia.io/test/conference.html (your own example on your own servers).

Similar things happen in our own environment, but I would really like that you have a working example first before trying to fix it on our end, as the problem likely lies in your JS SDK or on the backend server...

### Steps to reproduce
1.  Join a room with 2 desktop browsers streaming (can be same computer and webcam)
2.  Join the same room from a mobile device 
3. Make sure everyone sees each other
4.  Switch from Wifi -> 5G/4G, or the other way around.
    * Sometimes it might work fine, try switching network again if it doesn't trigger instantly...
 

### Expected behavior

After a network switch happens on mobile device, and access to internet "stabilizes", the following should happen:
- The mobile device should see the 2 other streams (coming from the computer)
- Mobile stream should continue streaming on the 2 desktop windows 

### Actual behavior
- Desktop streams stay black on mobile device (desktops see each other)
- Mobile stream doesn't start showing on desktop browsers again.
- Various errors about streamIdInUse
 
### Logs
No use since they are full of console log spam about `UPDATE_SOUND_LEVEL`

# Acceptance criteria

**For this bug to be considered fixed, the following should apply to your own `conference.html` example**:

With the same setup as in "Steps to reproduce", it should be possible to do at least 10 network changes on the mobile device (waiting for network to stabilize in between switches), and **all** streams should start showing on **all** clients **every time** after network has been switched...

**Comment by mekya** [comment]
I think it's related [to this PR ](https://github.com/ant-media/StreamApp/pull/468)

I've put it to the backlog and set the priority to high. 

Cheers

**Comment by MaZZly** [comment]
@mekya okay good if you already have a potential fix in the pipeline :)

**Comment by MaZZly** [comment]
@mustafaboleken @mekya what is the status and priority of that PR? It would be nice if this was working soon when we'll be back at work and need this working.

**Comment by mustafaboleken** [comment]
Hi @MaZZly,

The development is in the review process, let me talk with the team to merge it soon. Other than this one, we have active pr for solving the getStats issue and we added a new callback to notify others immediately if someone drops. As far as I remember these are the tasks that your team waits for.

**Comment by MaZZly** [comment]
@mustafaboleken  Perfect! I'll have a look once it is merged and NPM packages are updated :)

**Comment by MaZZly** [comment]
@mustafaboleken @mekya I just tested with `2.11.0-ALPHA-2024-Aug-2-13-17` which should have the above mentioned fix, and after the 3rd network switch on the mobile device, one of the video streams doesn't start up again.. 

Have you guys tested the above scenario on your own example code and it works when switching between Wifi and Mobile networks? (multiple times)?

**Comment by MaZZly** [comment]
@mustafaboleken @mekya  Actually, I made a mistake when updating to the Alpha version, which meant I was still on an older version. ( blame new OS install :see_no_evil: )

Now with `2.11.0-ALPHA-2024-Aug-2-13-17` installed (for real), it seems to be much more stable and I have not yet managed to trigger a case where all video streams would not reappear after a network change..

I will do some more testing to verify that it works, but it seems like the fix is working :sunglasses:  :+1:  

Sorry for the confusion, I should've double checked that I was really on the latest version.

**Comment by mekya** [comment]
Duplicate https://github.com/ant-media/Ant-Media-Server/issues/6333

**Comment by mustafaboleken** [comment]
Hi @MaZZly 

No worries, we are all human :) Please let us know about your test results.

**Comment by mekya** [comment]
Fix on the enterprise side

https://gitlab.com/Ant-Media/Ant-Media-Enterprise/-/merge_requests/726

**Comment by mekya** [comment]
The fixes are merged into the master branch and it'll be available in next release(2.11.1) likely in a couple of days. 

FYI
